src/main/java/sirius/db/mixing/query/QueryCompiler.java

```diff
@@ -710,7 +711,21 @@ protected Object compileStringValue(Property property, String value) {
             }
         }
 
-        return property.transformValue(Value.of(value));
+        // first line of defence: reference types are not processed as transformation triggers a lookup in the
+        // referenced database; if that lookup yields no results, the value is considered invalid despite the fact that
+        // looking for it in the original database may make sense
+        if (property instanceof BaseEntityRefProperty || property instanceof BaseEntityRefListProperty) {
+            return Value.of(value);
+        }
+
+        try {
+            return property.transformValue(Value.of(value));
+        } catch (Exception e) {
+            // second line of defence: should the transformation fail for whatever reason, we pass the original string
+            // as is, leaving it to the database to come up with no results if searching for an invalid value (such as a
+            // string in a numeric field)
+            return Value.of(value);
+        }
```

Comment:
Do we want to do this? Originally, when looking for a string in an integer field (for example), there was an error that the type is illegal. With the try/catch, we leave it to the database to handle that case.

Reply:
I'd keep this in place but also add "Exceptions.ignore(e)" this signals that the ex is deliberately ignore and also - for the desperate - there is a logger to enable und view these exception

Reply:
Done 👍